### DISCUSSION TITLE
Adding text to world

### DISCUSSION THREAD

**Discussion by normanhuettner** [dyn4j/dyn4j#193] [Q&A]
Hello William,

I am coding for my bachelor thesis in Psychology, that worked so far. However, I am not able to add text at any point of the world. How can this be added in general? I used the "UsingGraphics2D.java" file as starting point, so how would I add a string text into that world example?

Thank you in advance!
Norman

**Comment by wnbittle** [comment]
You should be able to this by adding to the render method something like:

g2d.drawString("text", x, y);
